packages/documentation-component/src/interfaces/Options.ts

```diff
@@ -0,0 +1,9 @@
+import { Plugin } from "./Plugin";
+import { Sources } from "./Source";
+import { RenderEngineWithOptions } from "./RenderEngine";
+
+export interface Options {
```

Comment:
this interface extends Context interface, use `extends` keyword

Reply:
Options has a sources field with Sources type, but Context has this same field with PureSources type :)